adcm_aio_client/core/objects/cm.py

```diff
@@ -85,6 +89,30 @@ def _main_prototype_id(self: Self) -> int:
 class BundlesNode(PaginatedAccessor[Bundle, None]):
     class_type = Bundle
 
+    async def _download_external_bundle(self: Self, url: UrlPath) -> bytes:
+        try:
+            async with httpx.AsyncClient() as client:
+                response = await client.get(url)
+                return response.content
+        except ValueError as err:
+            raise OperationError from err
+
+    async def create(self: Self, source: Path | UrlPath, accept_license: bool = False) -> Bundle:  # noqa: FBT001, FBT002
+        if isinstance(source, UrlPath):
+            file_content = await self._download_external_bundle(source)
+            files = {"file": file_content}
+        else:
+            files = {"file": Path(source).read_bytes()}
+
+        response = await self._requester.post("bundles", data=files)
+
+        bundle = Bundle(requester=self._requester, data=response.as_dict())
+
+        if accept_license:
+            bundle.license.accept()
```

Comment:
IMO accepting license should be "safe", so method should be called only when `unaccepted` is the license status